csharp-scenarios/01-Alien Translator.dib

```diff
@@ -0,0 +1,214 @@
+
+#!markdown
+
+For our first translation, let’s replace each language X word with its English word counterpart. To do this, we’ll use the [String.Replace method](https://docs.microsoft.com/dotnet/api/system.string.replace?view=net-6.0). To use this method correctly, we pass in the string we want to replace and the replacement string. Run the cell below to see how it works. 
+
+Here's an example:
+
+#!csharp
+
+String stringName = “He is cold”; 
+
+// Below, we replace "He" with "She" and "cold" with "cool"
+stringName.Replace("He”, "She”).Replace(“cold”,”cool”); 
+
+#!markdown
+
+>Notice how you can put multiple Replaces in one statement
+
+Now, let's try this out with the alien messsage.
+
+#!csharp
+
+//here’s the alien message 
+
+string message = "Warshel abree yoish";
+
+//try replacing the words in language x with their English counterparts using the Replace method 
+
+#!markdown
+
+## Let's use loops
+
+Looks like a second alien message is incoming:
+### Yoish shwaye ni ceroyash. Yoish oppor shioy abree coierm veiesht. Shir shioy abree yoish? 
+Replacing words one by one is a bit tedious for longer messages like this. We can make this process more efficient. Imagine if we could write a piece of code that goes through an unknown alien message word-by-word, finds the English translation for each word, and outputs the translated English message. Luckily we can by using loops. **Loops** are a way to repeat an action multiple times. 
+
+We'll use the ToEnglish **switch expression** above to translate each Language X word in the alien message to English.
+
+#!markdown
+
+Before we can loop over the words in the sentence, we need to break up the message string into words. To do this, we will use the [String.Split()](https://docs.microsoft.com/dotnet/api/system.string.split?view=net-6.0) method. We pass the String.Split() method a delineating character that the method will use to split the string into substrings. In the case of a sentence, we can choose a space ‘ ‘ as the delineating character because each word is separated by a space. In the cell below, use String.Split() to split up the sentence string into word substrings. 
+
+#!markdown
+
+![Split Method Image](https://raw.githubusercontent.com/katiesavage/csharp-notebooks/main/csharp-scenarios/images/split-method.png)
+
+#!markdown
+
+Now that we can split our sentence string into word substrings, it’s time to loop over them and translate! To do this, we’ll use a [foreach loop](https://docs.microsoft.com/dotnet/csharp/language-reference/statements/iteration-statements#the-foreach-statement). This is a special type of loop that executes a statement for each element in a group of that type of element. 
+
+The syntax works like this: 
+
+#!csharp
+
+string sentence = "this is an example";
+
+// split the string into substrings
+string[] variable_collection = sentence.Split(' ');
+
+foreach(string variable_name in variable_collection) 
+{ 
+     // statements to be executed 
```

Comment:
output variable_name.... I wouldn't use _ i would use variableCollection and variableName that is what we would use